clang/lib/StaticAnalyzer/Core/ExprEngineC.cpp

```diff
@@ -21,18 +21,19 @@ using namespace ento;
 using llvm::APSInt;
 
 /// Optionally conjure and return a symbol for offset when processing
-/// an expression \p Expression.
+/// a \p CFGElementRef.
```

Comment:
˙\p` should refer to a parameter name in the declaration. Consequently, this should be probably `ElemRef` currently.
But ideally, something like `Elem` or `Place` should be a better fit in general.